internal_docs/review-apps.md

```diff
@@ -0,0 +1,26 @@
+# Review apps for pull requests
+
+This repo uses [Heroku review apps](https://devcenter.heroku.com/articles/github-integration-review-apps)
+to give reviewers a way to easily preview and test changes to the kit.
+
+If you create a pull request from a fork of this repo (typical if the
```

Comment:
Minor, but the use of 'you' here doesn't seem quite right – authors of PRs from forks aren't the target audience for this doc, and if they were they won't have access to create a review app manually from the Heroku dashboard.

Suggest something like 'If a contributor opens a pull request from a fork'?